ReactWindows/ReactNative.Shared/Modules/I18N/I18NUtil.cs

```diff
@@ -32,11 +39,21 @@ public static bool IsRightToLeftAllowed
         {
             get
             {
+#if WINDOWS_UWP
                 return (bool?)ApplicationData.Current.LocalSettings.Values[AllowRTL] ?? false;
+#else
+                var result = default(bool);
+                var parsed = bool.TryParse(ConfigurationManager.AppSettings[AllowRTL], out result);
+                return (bool)(object)(parsed && result);
+#endif
```

Comment:
I started to do that but then didn't because it was going to make the file larger.